tests/my-first-test.spec.js

```diff
@@ -0,0 +1,26 @@
+import { test, expect } from '@playwright/test';
+import { exec } from 'child_process';
+
+const standardUser = "standard_user";
+const password = "secret_sauce";
+const url = 'https://www.saucedemo.com/';
```

Comment:
Lets move this to base url in playwright config, so you could set relative path into page.goto

Reply:
Fixed